asserts/asserts.go

```diff
@@ -138,6 +138,9 @@ func init() {
 	// 3: support for on-store/on-brand/on-model device scope constraints
 	// 4: support for plug-names/slot-names constraints
 	maxSupportedFormat[SnapDeclarationType.Name] = 4
+
+	// 1: support to limit to device serials
+	maxSupportedFormat[SystemUserType.Name] = 1
```

Comment:
we also need to add something to the formatAnalyzer table and make sure SuggestFormat works as expected. See similar code for snap-declaration.

Reply:
that means also checking that signing also doesn't with the expected error (see snap-declaration again)

Reply:
Thanks, I missed that indeed. Added now.